packages/core/src/components/spinner/_spinner.scss

```diff
@@ -211,7 +216,11 @@ $resources-path: "./resources" !default;
   }
 }
 
-@keyframes pt-spinner-animation {
-  from { transform: rotate(0deg); }
-  to   { transform: rotate(360deg); }
+.pt-input-group .pt-spinner {
```

Comment:
Ping on that. We declare `.pt-input-group .pt-icon`, `.pt-input-group .pt-tag` and `.pt-input-group .pt-button` in `_input-group.scss`, shouldn't be any different for `.pt-input-group .pt-spinner`